### PR TITLE
Docs: Trino Nessie configurations

### PR DESCRIPTION
**Pull request by ajantha-bhat**

### CODE REVIEW

File: site/docs/iceberg/trino.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Trino comes with Iceberg and Nessie catalog clients installed by default. 
+You can deploy Trino using any of the methods mentioned in [this document](https://trino.io/docs/current/installation.html) and to become familiar with the Iceberg connector, refer [this document](https://trino.io/docs/current/connector/iceberg.html?highlight=iceberg+connector).
+Nessie catalog properties required for the Iceberg connector can be found at https://trino.io/docs/current/object-storage/metastores.html#nessie-catalog.
+
+Sample Nessie configuration for Iceberg connector in `etc/catalog/iceberg.properties`
+```
+connector.name=iceberg
+iceberg.catalog.type=nessie
+iceberg.nessie-catalog.uri=http://localhost:19120/api/v2
+iceberg.nessie-catalog.default-warehouse-dir=/Users/foo/wh
```
Comment (XN137) [inline site/docs/iceberg/trino.md:16]: is this a hive/hadoop warehouse dir for local development?
wondering if an S3 based example would help more people (i.e. does it mean also need to configure fileio? i am not familiar with trino)

**Review by XN137 — DISMISSED** [review]
generally lgtm

File: site/docs/iceberg/trino.md (hunk 1 above)
Comment (ajantha-bhat) [reply site/docs/iceberg/trino.md:16]: Trino doesn't use Iceberg FileIO, we need to configure the credentials as mentioned in this section
https://trino.io/docs/current/connector/iceberg.html?highlight=s3+aws+access+key#file-system-access-configuration

File: site/docs/iceberg/trino.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+title: "Nessie + Iceberg + Trino"
+---
+
+# Trino via Iceberg
+
+Trino comes with Iceberg and Nessie catalog clients installed by default. 
+You can deploy Trino using any of the methods mentioned in [this document](https://trino.io/docs/current/installation.html) and to become familiar with the Iceberg connector, refer [this document](https://trino.io/docs/current/connector/iceberg.html?highlight=iceberg+connector).
```
Comment (dimas-b) [inline site/docs/iceberg/trino.md:8]: maybe `Trino can be deployed for use with Nessie by any of the methods mentioned in the [Trino Installation Guide](https://trino.io/docs/current/installation.html). To access Iceberg tables, the Iceberg connector is required. Refer to the [Trino Connector Documentation](*link*) for more information`?

**Review by dimas-b — COMMENTED** [review]
Nice doc addition. Thanks, @ajantha-bhat !

File: site/docs/iceberg/trino.md (hunk 1 above)
Comment (ajantha-bhat) [reply site/docs/iceberg/trino.md]: updated with s3 warehouse path.

File: site/docs/iceberg/trino.md (hunk 2 above)
Comment (ajantha-bhat) [reply site/docs/iceberg/trino.md]: done

**Comment by ajantha-bhat** [comment]
Anymore comments?

File: site/docs/iceberg/trino.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+title: "Nessie + Iceberg + Trino"
+---
+
+# Trino via Iceberg
+
+Trino can be deployed for use with Nessie by any of the methods mentioned in the [Trino Installation Guide](https://trino.io/docs/current/installation.html).
+To access Iceberg tables, need to configure the Iceberg connector. Refer to the [Trino Iceberg Connector Documentation](https://trino.io/docs/current/connector/iceberg.html?highlight=iceberg+connector) for more information.
```
Comment (dimas-b) [inline site/docs/iceberg/trino.md:8]: `need to ...` -> `one needs to ...`?

Comment (ajantha-bhat) [reply site/docs/iceberg/trino.md]: done

**Review by dimas-b — APPROVED** [review]

**Pull request merged**